src/Sylius/Bundle/ShopBundle/Resources/views/Common/Order/Label/PaymentState/orderPaymentState.html.twig

```diff
@@ -0,0 +1,19 @@
+{%
+    set viewOptions = {
+        authorized: { icon: "check icon", color: "orange" } ,
+        awaiting_payment: { icon: "clock icon", color: "grey" } ,
+        cancelled: { icon: "ban icon", color: "red" } ,
+        paid: { icon: "check icon", color: "green" } ,
+        partially_authorized: { icon: "check icon", color: "yellow" } ,
+        partially_paid: { icon: "adjust icon", color: "olive" } ,
+        partially_refunded: { icon: "reply icon", color: "violet" } ,
+        refunded: { icon: "reply all icon", color: "purple" } ,
+    }
+%}
+{% set value = 'sylius.ui.' ~ state %}
+
```

Comment:
```suggestion
```

Reply:
I think it is more readable with this space :)